### PR TITLE
Doc/basic example edits

### PR DESCRIPTION
**Pull request by PipKat**
Basic DPF Examples and Transient Analysis Examples

### CODE REVIEW

File: examples/00-basic/00-basic_example.py
Comment (akaszynski) [inline examples/00-basic/00-basic_example.py:7]:
```suggestion
basic post-processing.
```

Comment (akaszynski) [reply examples/00-basic/00-basic_example.py:7]: Replace all other occurrences of this.

Comment (PipKat) [reply examples/00-basic/00-basic_example.py:7]: @akaszynski Is there a reason for this. The Ansys Style Guideline says to use preprocessing and postprocessing. I can send you a link to the sharepoint site with all the guides if you'd like. However, I won't fight you on this if you really want it as two words as very few writers refer to these guidelines anyhow! Also, the check “Test Core API” is failing for both Windows and Linux failed and the error messages don't mean much to me. Sorry!

Comment (akaszynski) [reply examples/00-basic/00-basic_example.py:7]: No reason other than that's how it's written in pymapdl.  We'll have to fix it there to be in compliance with the Ansys Style Guideline.

Thanks for letting me know!

Comment (akaszynski) [reply examples/00-basic/00-basic_example.py:7]: > Also, the check “Test Core API” is failing for both Windows and Linux failed and the error messages don't mean much to me. Sorry!

That's related to a change with the latest release of pyvista.  I've capped the upper limit for pyvista for now; we'll fix that in a later release.

Comment (PipKat) [reply examples/00-basic/00-basic_example.py:7]: @akaszynski Can this PR be approved even though "Test Core API" is failing?

Comment (akaszynski) [reply examples/00-basic/00-basic_example.py:7]: > @akaszynski Can this PR be approved even though "Test Core API" is failing?

Meant to work on it this weekend.  I'd rather it pass so our merge commit history is clean.  Let me tweak something.

Comment (akaszynski) [reply examples/00-basic/00-basic_example.py:7]: Actually, CI is passing.

File: setup.py
Comment (cbellot000) [inline setup.py:67]: Hi @PipKat, why are we changing this version? Changing for < rather then >= means that we have to think about updating it regularly

Comment (akaszynski) [reply setup.py:67]: That's my doing.  It's related to our doc failures as commented https://github.com/pyansys/DPF-Core/pull/71#issuecomment-924973090.

There are four tests that will fail unless we cap at `pyvista<0.32` or modify the tests.  It's up to you if you want to modify the tests to get our CI/CD to pass or cap the version, but one will have to be done.

Comment (anslpa) [reply setup.py:67]: @akaszynski @cbellot000 @PipKat I think this is better to update the tests (done in the following PR: #73), and keep the pyvista requirement versioning as it is right now.

File: requirements_docs.txt
Comment (akaszynski) [inline requirements_docs.txt:3]:
```suggestion
pyvista
```

File: requirements_test.txt
Comment (akaszynski) [inline requirements_test.txt:5]:
```suggestion
pyvista>=0.24.0
```

File: setup.py
Comment (akaszynski) [inline setup.py:67]:
```suggestion
        "plotting":  ['pyvista>=0.24.0', 'matplotlib==3.2'],
```

File: setup.py
Comment (akaszynski) [reply setup.py]: I've reverted the changes.  We're going to have to wait for #73 to be merged to validate the CI.  I'll let you merge it (never liked it when other people merged my PRs :)

**Review by akaszynski — APPROVED** [review]
Thanks @PipKat.  We've fixed the tests and these are good to go.

**Pull request merged**
